src/parser.js

```diff
@@ -251,6 +251,9 @@ export default class Parser extends Stream {
               if (entry.attributes['INSTREAM-ID']) {
                 rendition.instreamId = entry.attributes['INSTREAM-ID'];
               }
+              if (entry.attributes['CHARACTERISTICS']) {
```

Comment:
that's why the build is failing. `INSTREAM-ID` requires the bracket notation because of the hyphen; if you take a look at other things, they use dot-notation.